### PR TITLE
[PAN-3132] Fix bpk nudger for subtitle long case

### PR DESCRIPTION
**Pull request by noah-wen**
### Screenshot
| Old| New|
| ----| ----|
| image| image|

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt [hunk 1]
```diff
@@ -160,8 +160,8 @@ fun BpkNudger(
                         text = subtitle,
                         style = BpkTheme.typography.bodyDefault,
                         color = BpkTheme.colors.textSecondary,
-                        maxLines = 1,
-                        overflow = TextOverflow.Ellipsis,
+                        maxLines = Int.MAX_VALUE,
+                        overflow = TextOverflow.Clip,
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt:164]: nit: both lines can be removed as we're just using the default values

**Review by marianeum — COMMENTED** [review]
can you please add a snapshot test for the long line variant?

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt (hunk 1 above)
Comment (noah-wen) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt:164]: Oh, I just see.

Comment (noah-wen) [reply backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/nudger/BpkNudger.kt:164]: Yeah, It upgraded.

File: app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt [hunk 2]
```diff
@@ -72,6 +72,11 @@ fun NudgerStory(modifier: Modifier = Modifier) {
             modifier = Modifier.fillMaxWidth(),
         )
 
+        NudgerRowExample(
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt:75]: this is good too, but what i meant is a snapshot test (so `BpkNudgerTest`)

File: app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt [hunk 3]
```diff
@@ -72,6 +72,11 @@ fun NudgerStory(modifier: Modifier = Modifier) {
...
+            modifier = Modifier.fillMaxWidth(),
+            subtitle = "subtitle is an example of long text that can be wrapped",
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt:77]: we generally use string resources - there's probably already a generic one that can be reused)

Comment (noah-wen) [reply app/src/main/java/net/skyscanner/backpack/demo/compose/NudgerStory.kt]: Right! Have updating with above comment too, could you help to take a look again? Thanks!

**Review by marianeum — APPROVED** [review]
thanks - looks good! when the snapshots are re-generated please push an empty commit and then ping me to merge

**Comment by noah-wen** [comment]
> thanks - looks good! when the snapshots are re-generated please push an empty commit and then ping me to merge

Hey @marianeum , seems snapshots updated, I have push the empty commit just now.

**Pull request merged**